# papers with data to mine

There are a lot of published papers with a lot of data and mining data can make new discoveries.

### CCLE 

* [Quantitative Proteomics of the Cancer Cell Line Encyclopedia](https://www.cell.com/cell/fulltext/S0092-8674(19)31385-6)
* [Pan-cancer single cell RNA-seq uncovers recurring programs of cellular heterogeneity](https://www.biorxiv.org/content/10.1101/807552v1.full)
* [Next-generation characterization of the Cancer Cell Line Encyclopedia](https://www.nature.com/articles/s41586-019-1186-3)
Broad has [DepMap](https://depmap.org/portal/)
* [Pan-cancer single-cell RNA-seq identifies recurring programs of cellular heterogeneity](https://www.nature.com/articles/s41588-020-00726-6) scRNAseq for CCLE lines!

[depmap bioc package](http://bioconductor.org/packages/release/data/experiment/html/depmap.html)

### TCGA & ICGC

* [The chromatin accessibility landscape of primary human cancers](https://science.sciencemag.org/content/362/6413/eaav1898)
* [Disruption of chromatin folding domains by somatic genomic rearrangements in human cancer](https://www.nature.com/articles/s41588-019-0564-y)
* [Pan-cancer analysis of whole genomes](https://www.nature.com/articles/s41586-020-1969-6)

### immune 
* [The cis-Regulatory Atlas of the Mouse Immune System](https://www.cell.com/cell/fulltext/S0092-8674(18)31650-7)

### chromatin interaction
* [3D Genome Browser from Yue Lab](http://promoter.bx.psu.edu/hi-c/)

### chromatin accessibility

* [High-resolution genetic mapping of putative causal interactions between regions of open chromatin](https://www.nature.com/articles/s41588-018-0278-6)

* [Global reference mapping and dynamics of human transcription factor footprints](https://www.biorxiv.org/content/10.1101/2020.01.31.927798v1) We map the fine-scale structure of ~1.6 million DHS

* [Index and biological spectrum of accessible DNA elements in the human genome](https://www.biorxiv.org/content/10.1101/822510v2) Download the data from https://www.meuleman.org/project/dhsindex/ Wouter Meuleman is the author of epilogos.

### epigenome annotation

* [Integrative analysis of 10,000 epigenomic maps across 800 samples for regulatory genomics and disease dissection](https://www.biorxiv.org/content/10.1101/810291v2) by Manolis Kellis group: To help elucidate genetic variants underlying complex traits, we develop EpiMap, a compendium of 833 reference epigenomes across 18 uniformly-processed and computationally-completed assays. We define chromatin states, high-resolution enhancers, activity patterns, enhancer modules, upstream regulators, and downstream target gene functions.


### single cell

* [Single-Cell Transcriptome Atlas of Murine Endothelial Cells](https://www.cell.com/cell/pdf/S0092-8674(20)30062-3.pdf)
* [A cell atlas of human thymic development defines T cell repertoire formation](https://science.sciencemag.org/content/367/6480/eaay3224)
* [Chromatin accessibility dynamics in a model of human forebrain development](https://science.sciencemag.org/content/367/6476/eaay1645) Trevino et al. avoided this problem by developing three-dimensional organoid models of human forebrain development and examining chromatin accessibility and gene expression at the single-cell level.
* [Single-cell transcriptome profiling an adult human cell atlas of 15 major organs](https://www.biorxiv.org/content/10.1101/2020.03.18.996975v1)

"we performed single-cell transcriptomes of 88,622 cells derived from 15 tissue organs of one adult donor and generated an adult human cell atlas (AHCA). The AHCA depicted 234 subtypes of cells, including major cell types such as T, B, myeloid, epithelial, and stromal cells, as well as novel cell types in skin, each of which was distinguished by multiple marker genes and transcriptional profiles and collectively contributed to the heterogeneity of major human organs. Moreover, TCR and BCR repertoire comparison and trajectory analyses revealed direct clonal sharing of T and B cells with various developmental states among different tissues. Furthermore, novel cell markers, transcription factors and ligand-receptor pairs were identified with potential functional regulations on maintaining the homeostasis of human cells among tissues."

* [Construction of a human cell landscape at single-cell level](https://www.nature.com/articles/s41586-020-2157-4) database url http://bis.zju.edu.cn/HCL/
