# papers with data to mine

There are a lot of published papers with a lot of data and mining data can make new discoveries.

### review 

[Big data in basic and translational cancer research](https://www.nature.com/articles/s41568-022-00502-0)

### 

* [Integrating and formatting biomedical data as pre-calculated knowledge graph embeddings in the Bioteque](https://www.nature.com/articles/s41467-022-33026-0) Here we present the Bioteque, a resource of unprecedented size and scope that contains pre-calculated biomedical descriptors derived from a gigantic knowledge graph, displaying more than 450 thousand biological entities and 30 million relationships between them. The Bioteque integrates, harmonizes, and formats data collected from over 150 data sources, including 12 biological entities (e.g., genes, diseases, drugs) linked by 67 types of associations (e.g., ‘drug treats disease’, ‘gene interacts with gene’)

### proteomics

* [CPTAC python package](https://github.com/PayneLab/cptac) The National Cancer Institute’s Clinical Proteomic Tumor Analysis Consortium (CPTAC) is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics. CPTAC generates comprehensive proteomics and genomics data from clinical cohorts, typically with ~100 samples per tumor type. The graphic below summarizes the structure of each CPTAC dataset.
* [Pan-cancer proteomic map of 949 human cell lines](https://www.sciencedirect.com/science/article/pii/S1535610822002744) download https://cellmodelpassports.sanger.ac.uk/
* [Quantitative Proteomics of the Cancer Cell Line Encyclopedia](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7339254/)

### methylation 

recount methylation: https://bioconductor.org/packages/release/bioc/vignettes/recountmethylation/inst/doc/recountmethylation_users_guide.html

### CCLE 

* [Quantitative Proteomics of the Cancer Cell Line Encyclopedia](https://www.cell.com/cell/fulltext/S0092-8674(19)31385-6)
* [Pan-cancer single cell RNA-seq uncovers recurring programs of cellular heterogeneity](https://www.biorxiv.org/content/10.1101/807552v1.full)
* [Next-generation characterization of the Cancer Cell Line Encyclopedia](https://www.nature.com/articles/s41586-019-1186-3)
Broad has [DepMap](https://depmap.org/portal/)
* [Pan-cancer single-cell RNA-seq identifies recurring programs of cellular heterogeneity](https://www.nature.com/articles/s41588-020-00726-6) scRNAseq for CCLE lines!
* [A metastasis map of human cancer cell lines](https://pubmed.ncbi.nlm.nih.gov/33299191/)

[depmap bioc package](http://bioconductor.org/packages/release/data/experiment/html/depmap.html)
https://depmap.org/portal/download/custom/
https://depmap.org/portal/interactive/

### TCGA & ICGC

* [The chromatin accessibility landscape of primary human cancers](https://science.sciencemag.org/content/362/6413/eaav1898)
* [Disruption of chromatin folding domains by somatic genomic rearrangements in human cancer](https://www.nature.com/articles/s41588-019-0564-y)
* [Pan-cancer analysis of whole genomes](https://www.nature.com/articles/s41586-020-1969-6)
* [Removing unwanted variation from large-scale RNA sequencing data with PRPS](https://www.nature.com/articles/s41587-022-01440-w) normalize TCGA data 

### UCSC 

UCSC xena https://xena.ucsc.edu/

### UKbiobank

* [200,000 whole genomes made available for biomedical studies by U.K. effort](https://www.science.org/content/article/200-000-whole-genomes-made-available-biomedical-studies-uk-effort) We present mutational signature analyses of 12,222 whole-genome–sequenced cancers collected prospectively via the UK National Health Service (NHS) for the 100,000 Genomes Project

* [Substitution mutational signatures in whole-genome–sequenced cancers in the UK population](https://www.science.org/doi/10.1126/science.abl9283)

### PDX

* [Comprehensive characterization of 536 patient-derived xenograft models prioritizes candidatesfor targeted treatment](https://www.nature.com/articles/s41467-021-25177-3#data-availability) https://portal.pdxnetwork.org/

### immune 
* [The cis-Regulatory Atlas of the Mouse Immune System](https://www.cell.com/cell/fulltext/S0092-8674(18)31650-7)

### chromatin interaction
* [3D Genome Browser from Yue Lab](http://promoter.bx.psu.edu/hi-c/)

### chromatin accessibility

* [High-resolution genetic mapping of putative causal interactions between regions of open chromatin](https://www.nature.com/articles/s41588-018-0278-6)

* [Global reference mapping and dynamics of human transcription factor footprints](https://www.biorxiv.org/content/10.1101/2020.01.31.927798v1) We map the fine-scale structure of ~1.6 million DHS

* [Index and biological spectrum of accessible DNA elements in the human genome](https://www.biorxiv.org/content/10.1101/822510v2) Download the data from https://www.meuleman.org/project/dhsindex/ Wouter Meuleman is the author of epilogos.

### epigenome annotation

* [Integrative analysis of 10,000 epigenomic maps across 800 samples for regulatory genomics and disease dissection](https://www.biorxiv.org/content/10.1101/810291v2) by Manolis Kellis group: To help elucidate genetic variants underlying complex traits, we develop EpiMap, a compendium of 833 reference epigenomes across 18 uniformly-processed and computationally-completed assays. We define chromatin states, high-resolution enhancers, activity patterns, enhancer modules, upstream regulators, and downstream target gene functions.

* [Boix et. al bioRxiv (2019) - Integrative analysis of 10,000 epigenomic maps across 800 samples for regulatory genomics and disease dissection](https://www.biorxiv.org/content/10.1101/810291v2) http://compbio.mit.edu/epimap/

### structural variations

* [Haplotype-resolved diverse human genomes and integrated analysis of structural variation](https://science.sciencemag.org/content/early/2021/02/24/science.abf7117)

### single cell

* [Single-Cell Transcriptome Atlas of Murine Endothelial Cells](https://www.cell.com/cell/pdf/S0092-8674(20)30062-3.pdf)
* [A cell atlas of human thymic development defines T cell repertoire formation](https://science.sciencemag.org/content/367/6480/eaay3224)
* [Chromatin accessibility dynamics in a model of human forebrain development](https://science.sciencemag.org/content/367/6476/eaay1645) Trevino et al. avoided this problem by developing three-dimensional organoid models of human forebrain development and examining chromatin accessibility and gene expression at the single-cell level.
* [Single-cell transcriptome profiling an adult human cell atlas of 15 major organs](https://www.biorxiv.org/content/10.1101/2020.03.18.996975v1)

"we performed single-cell transcriptomes of 88,622 cells derived from 15 tissue organs of one adult donor and generated an adult human cell atlas (AHCA). The AHCA depicted 234 subtypes of cells, including major cell types such as T, B, myeloid, epithelial, and stromal cells, as well as novel cell types in skin, each of which was distinguished by multiple marker genes and transcriptional profiles and collectively contributed to the heterogeneity of major human organs. Moreover, TCR and BCR repertoire comparison and trajectory analyses revealed direct clonal sharing of T and B cells with various developmental states among different tissues. Furthermore, novel cell markers, transcription factors and ligand-receptor pairs were identified with potential functional regulations on maintaining the homeostasis of human cells among tissues."

* [Construction of a human cell landscape at single-cell level](https://www.nature.com/articles/s41586-020-2157-4) database url http://bis.zju.edu.cn/HCL/
* [A human cell atlas of fetal chromatin accessibility](https://science.sciencemag.org/content/370/6518/eaba7612)
* [A comprehensive library of human transcription factors for cell fate engineering](https://www.nature.com/articles/s41587-020-0742-6)
* [A single–cell type transcriptomics map of human tissues](https://www.science.org/doi/10.1126/sciadv.abh2169#.YTis17ZVXh8.twitter)
* [Signatures of plasticity, metastasis, and immunosuppression in an atlas of human small cell lung cancer](https://www.cell.com/cancer-cell/fulltext/S1535-6108(21)00497-9)
* [Integrated single-cell transcriptomics and epigenomics reveals strong germinal center–associated etiology of autoimmune risk loci](https://www.science.org/doi/10.1126/sciimmunol.abh3768)
* [A single-cell atlas of chromatin accessibility in the human genome](https://www.cell.com/cell/fulltext/S0092-8674(21)01279-4) from Bing Ren group.
* [The Tabula Sapiens: a multiple organ single cell transcriptomic atlas of humans](https://www.biorxiv.org/content/10.1101/2021.07.19.452956v2) We used single cell transcriptomics to create a molecularly defined phenotypic reference of human cell types which spans 24 human tissues and organs." The authors also systematically analyzed splicing differences at the single-cell resolution. For interactive visulization: https://tabula-sapiens-portal.ds.czbiohub.org/splicing
  
### image data

* cellpainting gallary from Broad https://registry.opendata.aws/cellpainting-gallery/

* Bioimagingguide.org, a companion website to paper [“A biologist’s guide to planning and performing quantitative bioimaging experiments”](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3002167)
  
* Tutorials.cellprofiler.org – a list of written tutorials, many with accompanying videos

* [Data-analysis strategies for image-based cell profiling](https://www.nature.com/articles/nmeth.4397) – the gold-standard methods paper for performing image-based profiling

* [Interpreting Image-based Profiles using Similarity Clustering and Single-Cell Visualization](https://currentprotocols.onlinelibrary.wiley.com/doi/10.1002/cpz1.713) – a protocol paper on how to interpret image-based profiling experiments once you’ve performed them
